app/models/question.rb

```diff
@@ -48,7 +48,7 @@ class Question < ApplicationRecord
   class_attribute :model_exporter, default: nil, instance_writer: false
   class_attribute :blackboard_export_type, default: nil, instance_writer: false
   class_attribute :moodle_type, default: nil, instance_writer: false
```

Comment:
not related to this PR but noting that we should change moodle_type to moodle_export_type to match the others. maybe we can do that in this [ticket](https://github.com/notch8/viva/issues/403)?

Reply:
Yeah I planned to look at consistency in naming